metadrive/policy/replay_policy.py

```diff
@@ -94,26 +53,44 @@ def act(self, *args, **kwargs):
             return None  # Return None action so the base vehicle will not overwrite the steering & throttle
 
         if "throttle_brake" in info:
-            self.control_object.set_throttle_brake(float(info["throttle_brake"]))
+            if hasattr(self.control_object, "set_throttle_brake"):
+                self.control_object.set_throttle_brake(float(info["throttle_brake"]))
```

Comment:
should raise error here. Why store throttle for an object that can't be set? 

If we can't set the throttle that are stored in the data, it means that we are making some deviation from the data, which is wrong and should yield error.

Reply:
Cyclist and Pedestrian has no this function, but still contain throttle_brake